app/templates/_package.json

```diff
@@ -14,7 +14,9 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "xo && <% if (nyc) { %>nyc <% } %>ava"
+    "test": "xo <% if (babel) { %>src/*.js test*.js && npm run babel <% } %>&& <% if (nyc) { %>nyc <% } %>ava"<% if (babel) { %>,
+    "babel": "babel src --out-dir=. --presets=es2015",
```

Comment:
I think @novemberborn uses `.` over `dist`. Maybe he can argue the counterpoint.